java/client/src/main/java/glide/api/models/commands/SetOptions.java

```diff
@@ -0,0 +1,129 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.models.exceptions.RequestException;
+import java.util.LinkedList;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import redis_request.RedisRequestOuterClass.Command;
+
+/**
+ * Optional arguments for StringCommands#set(String, String, SetOptions) command.
+ *
+ * @see <a href="https://redis.io/commands/set/">redis.io</a>
+ */
+@Builder
+public final class SetOptions {
```

Comment:
can this be remove from this PR?
You can add it after the set PR is merged

Reply:
I was actually intending to merge this _after_ the set() command PR.  There's no rush on this one.